packages/surveys/src/components/questions/OpenTextQuestion.tsx

```diff
@@ -109,6 +111,8 @@ export const OpenTextQuestion = ({
                 className="fb-border-border placeholder:fb-text-placeholder fb-text-subheading focus:fb-border-brand fb-bg-input-bg fb-rounded-custom fb-block fb-w-full fb-border fb-p-2 fb-shadow-sm focus:fb-outline-none focus:fb-ring-0 sm:fb-text-sm"
                 pattern={question.inputType === "phone" ? "[0-9+ ]+" : ".*"}
                 title={question.inputType === "phone" ? "Enter a valid phone number" : undefined}
+                minlength={question.inputType === "text" ? question.minLength || undefined : undefined}
+                maxlength={question.inputType === "text" ? question.maxLength || undefined : undefined}
```

Comment:
The value of `question.minLength` or `question.maxLength` is already undefined if unset. This `OR` value is not required. Can you please remove that?